assignment/index.html

```diff
@@ -62,11 +62,30 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
+      var jsonToCsv = function(json) {
```

Comment:
This output of this task is "csv-style" array of arrays that turns:

`[{'first': 1, 'second': 44}, {'first': 2, 'second': 12}]`
       into

 `[['first','second'],[1, 44],[2, 12]]`